src/templates/ssr/fabric-custom-xl/index.ts

```diff
@@ -0,0 +1,67 @@
+import { post } from '$lib/messenger';
+
+const CACHE_BUST = '%%CACHEBUSTER%%';
+const DapAssetsRoot = `https://s3-eu-west-1.amazonaws.com/adops-assets/dap-fabrics`;
+const DapAssetsFolder: string = '[%DapAssetsFolder%]';
+
+const DapAssetsPath = `${DapAssetsRoot}/${DapAssetsFolder}`;
+const ThirdPartyTag: string = '[%ThirdPartyTag%]';
```

Comment:
You removed ThirdPartyTag for the Fabric Custom - should it be included here or should it be removed from this template too?

Reply:
Ah yes good shout, thanks, Ill add to the description as well to keep a note of this going forward 👍 
There seems to be a difference between  `thirdPartyJSTracking` and `ThirdPartyTag`which are unrelated.